### ISSUE TITLE
Note Section/System for case workers/managers

### ISSUE THREAD

**Issue by leekahung** [codeforpdx/PASS#398]
### Feature Title:
Note Section/System for case workers/managers

---
### Feature Description/Use Case:
One of the features mentioned in our discussions with case workers/managers is that it'll be good to have a place where they could store notes of clients. While I'm not sure how this feature might be duplicated in the case workers/managers workflow with clients since they have a similar ability to record notes via HMIS, having an area to write on store notes on a pod might be beneficial, especially with regards to notes on documents for example.

---
### Proposed Solution:
A potential solution to this is to create a new field/section for contact notes similar to how it's done with `Contacts` and `Profile` pages. A link in the `contact list` can be used to bring users to the contact's profile information and documents along with a new section for `Notes` in their `Profile` page.

**Comment by andycwilliams** [comment]
Given the potential refocusing of PASS's core requirements, are we still looking to add this as part of MVP?

**Comment by leekahung** [comment]
> Given the potential refocusing of PASS's core requirements, are we still looking to add this as part of MVP?

If we're refocusing PASS to just storage and address book, think this could be dropped.

**Comment by andycwilliams** [comment]
Yeah, I think we should revisit this down the line but for the moment the scope of the project seems narrower.
